src/Core/Tool/Authorizer/UserAuthorizer.php

```diff
@@ -85,11 +85,6 @@ public function isAllowed(Entity $entity, $privilege)
 			return true;
 		}
 
-		// Regular user can always read
```

Comment:
in line 84, where they are checking 		if ($this->isUserSelf($entity) && in_array($privilege, ['update', 'read_full',])) {
we should add 'read' to the list. Users should see their own data